js/src/components/modals/KeyphrasesTable.js

```diff
@@ -0,0 +1,44 @@
+/* External dependencies */
+import React from "react";
+import PropTypes from "prop-types";
+import { Fragment } from "@wordpress/element";
+
+/**
+ * The Related Keyphrases table component.
+ */
+class KeyphrasesTable extends React.Component {
+	/**
+	 * Constructs the Related Keyphrases table.
+	 *
+	 * @param {Object} props The props for the Related Keyphrases table.
+	 *
+	 * @returns {void}
+	 */
+	constructor( props ) {
+		super( props );
+	}
+
+	/**
+	 * Renders the Related Keyphrases table.
+	 *
+	 * @returns {ReactElement} The Related Keyphrases table.
+	 */
+	render() {
+		return (
+			<Fragment>
+				<h2 id={ this.props.fakeProp }>Keyphrase table</h2>
```

Comment:
Why use a fake prop / what is the added value of this prop? Perhaps add some temporary documentation to clear up what's happening.